dataset/resize.py

```diff
@@ -0,0 +1,67 @@
+from os import listdir
+import cv2
+from os.path import join, isfile, isdir
+
+
+class VideoResizer:
+    def __init__(self, source_dir, dest_dir):
+        self.source_dir = source_dir
+        self.dest_dir = dest_dir
+
+    def resize_video(self, input_file, output_file, size):
+        video = cv2.VideoCapture(input_file)
+        fps = video.get(cv2.CAP_PROP_FPS)
+        fourcc = cv2.VideoWriter_fourcc(*'XVID')
+        out = cv2.VideoWriter(output_file, fourcc, fps, size)
+
+        # iterate through each frame
+        while video.isOpened():
+            ret, frame = video.read()
+            if not ret:
+                break
+            # use built in resize function - need to be changed
```

Comment:
we should first select what is the greater value out of height and width, then accordingly resize the frame including padding, where it's needed.

Reply:
Yeah.. That's why I mentioned need to be changed. Will change this soon.